### PR TITLE
Implemented bfs on a tree

### PR DESCRIPTION
**Pull request by JSimoni42**
- Wrote a prompt with example tree
- Created Node class for building trees
- Hardcoded example tree
- Implemented BFS for trees using queue

### CODE REVIEW

File: bfs_on_tree/bfsOnTree.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      }
+      if (current.left != null) {
+        q.add(current.left);
+      }
+      if (current.right != null) {
+        q.add(current.right);
+      }
+      current = q.poll();
+    }
+    return true;
```
Comment (paulliu87) [inline bfs_on_tree/bfsOnTree.java]: I think this would be return false outside the while loop.

**Comment by paulliu87** [comment]
👍  🐙  👍  first PR!!!!!!!!!

**Comment by JSimoni42** [comment]
Good catch!

**Comment by paulliu87** [comment]
I also invite you to the Cogrammers channel.

**Pull request merged**